experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/objectFactory.ts

```diff
@@ -32,8 +23,9 @@ export function addFactory<TSchema extends ObjectNodeSchema>(
 		const node = Object.create(null);
 		// Shallow copy the content and then add the type name symbol to it.
 		// The copy is necessary so that the input `content` object can be re-used as the contents of a different typed/named node in another `create` call.
-		const namedContent = { ...content, [typeNameSymbol]: schema.name };
-		Object.defineProperty(node, factoryContent, { value: namedContent });
+		const contentCopy = { ...content };
+		Object.defineProperty(contentCopy, typeNameSymbol, { value: schema.name });
```

Comment:
Nit: might be worth adding a quick comment explaining why we need to use the `defineProperty` pattern for these symbols, since it isn't a common pattern.

Reply:
(FYI: the answer is to be non-enumerable, non-configurable, non-writable.)